core/trace/dt_span_serializer.go

```diff
@@ -254,16 +254,15 @@ func getSerializedResourceForSpanExport(spanResource *resource.Resource) ([]byte
 	if err != nil {
 		return nil, err
 	}
-	resource := protoResource.Resource{
+	res := protoResource.Resource{
 		Attributes: protoAttributes,
 	}
-	return proto.Marshal(&resource)
+	return proto.Marshal(&res)
 }
 
 func getExporterResource() *resource.Resource {
 	return resource.NewSchemaless(
-		// TODO get the attribute keys from the generated semantic conventions
-		attribute.Key("telemetry.exporter.name").String("odin"),
-		attribute.Key("telemetry.exporter.version").String(version.FullVersion),
+		attribute.Key(semconv.TelemetryExporterName).String("odin"),
```

Comment:
```suggestion
		attribute.Key(semconv.TelemetryExporterName).String(semconv.TelemetryExporterNameOdin),
```

There is a constant for that as well.